result/yoda.js

```diff
@@ -0,0 +1,6 @@
+
+var foo = {
```

Comment:
__[development][eslint]__
* Error - 'foo' is defined but never used (no-unused-vars)